File: pytools/examples/java/client/protocol/AckSceneEnter.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckSceneEnter
{
	private MsgScenePlayer player;


	public AckSceneEnter(Packet packet)
	{
		player = new MsgScenePlayer(packet);
	}



	public MsgScenePlayer getPlayer()
	{
		return this.player;
	}

}

File: pytools/data/data.code.php.client/AckSceneEnter.php
<?php
namespace protocol;

use network\Packet;


class AckSceneEnter
{
	private $player = null;


	public function __construct(&$packet)
	{
		$this->decode($packet);
	}

	public function decode(&$packet)
	{
		$this->player = new MsgScenePlayer($packet);
	}


	public function getPlayer()
	{
		return $this->player;
	}

}
